src/sentry/analytics/events/admin_request.py

```diff
@@ -0,0 +1,16 @@
+from sentry import analytics
+
+
+class InviteRequestSent(analytics.Event):
+    type = "admin_request.sent"
+    attributes = (
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("user_id"),
+        analytics.Attribute("target_user_id"),
+        analytics.Attribute("providers"),
+        analytics.Attribute("ty"),
```

Comment:
@Neo-Zhixing What does `ty` represent?

Reply:
ty means type. I had problems naming it `type` so I just used an acronym.